data/virt_autotest/kubevirt_tests_config_parser.pl

```diff
@@ -0,0 +1,21 @@
+use strict;
+use warnings;
+use Config::Tiny;
+
+my $config    = Config::Tiny->read($ARGV[0]);
+my $go_test   = '';
+my $skip_test = '';
+my $extra_opt = '';
+
+foreach my $key (keys %{$config->{$ARGV[1]}}) {
+    if ($key eq 'GINKGO_FOCUS') {
+        $go_test = $config->{$ARGV[1]}->{$key};
+    } elsif ($key eq 'GINKGO_SKIP') {
+        $skip_test = "|$config->{$ARGV[1]}->{$key}";
+    } elsif ($key eq 'EXTRA_OPT') {
+        $extra_opt = $config->{$ARGV[1]}->{$key};
+    }
+}
+
+my $params = join(",", $go_test, $skip_test, $extra_opt);
+print $params;
```

Comment:
Could you help put kubevirt_tests_config_parser.pl and kubevirt-tests.conf into a subfolder kubevirt_tests in data/virt_autotest ?

Reply:
```
 print "$ARGV[1]:$params";
```
Better ?

Reply:
> Could you help put kubevirt_tests_config_parser.pl and kubevirt-tests.conf into a subfolder kubevirt_tests in data/virt_autotest ?

Will create and move them into a separate directory.